app/main.py

```diff
@@ -2,24 +2,67 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(
+            self,
+            min_amount: int,
+            max_amount: int
+    ) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, instance: "Visitor", name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: "Visitor", owner: object) -> object:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: "Visitor", value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Expected an integer value")
+        if not (self.min_amount <= value <= self.max_amount):
+            raise ValueError("Value out of the allowed range")
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
The `Visitor` class is a simple data class that holds information about a visitor. It might be a good idea to add some data validation in the `__init__` method to ensure that the `name` is a string and that `age`, `weight`, and `height` are integers.